### PR TITLE
Include.js changes

### PR DESCRIPTION
**Pull request by Hrishap**
Recommended changes  made in include.js

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -610,6 +610,7 @@ let cards = [
     githubLink: 'https://github.com/migueldalberto'
   },
   {
+
```
Comment (sophiabrandt) [inline include.js:613]: please don't reformat existing code. An empty line is not necessary here.

File: include.js [hunk 2]
```diff
@@ -1114,13 +1115,22 @@ let cards = [
...
   },
 
+  {
+    artName: 'Background line animation',
+    pageLink: './Art/Hrishap/index.html',
+    imageLink: './Art/Hrishap/Background line animation.gif',
+    author: 'Hrishap',
+    githubLink: 'https://github.com/Hrishap'
+
+
```
Comment (sophiabrandt) [inline include.js:1133]: This is missing the closing bracket and comma `},`. Also, please no empty lines between entries.

File: include.js [hunk 3]
```diff
@@ -1282,7 +1292,9 @@ let cards = [
     imageLink: './Art/vivek/waves.gif',
     author: 'Vivek Boii',
     githubLink: 'https://github.com/vivekBoii'
+
```
Comment (sophiabrandt) [inline include.js:1295]: same here.

File: include.js [hunk 4]
```diff
@@ -1282,7 +1292,9 @@ let cards = [
...
   }
+
```
Comment (sophiabrandt) [inline include.js:1297]: same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by Hrishap** [comment]
respected changes done :)

**Review by MattCSmith — APPROVED** [review]
Looks good to me. Nice job 👍

**Pull request merged**